src/System.Net.Sockets/src/RefactoringShims/InteropShim.cs

```diff
@@ -0,0 +1,62 @@
+// Copyright (c) Microsoft. All rights reserved.
```

Comment:
I can't find references to this in the Linux_Debug configuration. Is this still needed?